muttlib.c

```diff
@@ -76,6 +76,18 @@ static const char *xdg_defaults[] =
   [kXDGConfigDirs] = "/etc/xdg",
 };
 
+static const char * const mailcap_text_as_application[] =
+{
+  "html",
+  "xml",
+  "calendar",
+};
```

Comment:
I think this should be defined as a `LIST*` exposed as a global `optiont_t` in `init.h`, with a `set` or a command to update that list.